salt/files/minion.d/f_defaults.conf

```diff
@@ -43,6 +43,12 @@ master:
 # choose a random master
 {{ get_config('random_master', 'False') }}
 
+# Set the master type
+{{ get_config('master_type', 'str') }}
```

Comment:
I was very confused about this because of the differences between different doc pages and the comments in the files. I can see it's going to change, but it appears it's 'str' in the current release.

https://github.com/saltstack/salt/blob/v2015.5.2/salt/config.py#L289

So is it better to go with the current release, or the future value? Or perhaps the answer is if the pillar says either "str" or "standard", don't put a value in.